options/locale/locale_en-US.ini

```diff
@@ -578,6 +578,10 @@ email_notifications.submit = Set Email Preference
 owner = Owner
 repo_name = Repository Name
 repo_name_helper = Good repository names use short, memorable and unique keywords.
+template = Template
+template_select = Select a template.
+template_helper = Make repository a template
+template_description = A repository template can be used to generate a repository with the same files as the template.
```

Comment:
This text should have a broader description, as `git contents` is actually optional and other options are being considered.

Reply:
`A repository template can be used to generate a repository based on this one.`

I am open to other wording, I'm not sure I 100% like this wording either.

Reply:
Github uses "Template repositories let users generate new repositories with the same directory structure and files"

Maybe this could be "Template repositories let users generate new repositories with the same directory structure, files, and optional settings"

Reply:
Updated to @mrsdizzie recommendation. I think it sounds much better.